CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
 
+## 1.13.1 - TBD
```

Comment:
We can probably drop `CHANGELOG.md` - it's more messy than useful (especially for merge-ups)

Reply:
Okay.. i was have trouble trying to resign commits by originally created by the bot.

Reply:
I think the entire `CHANGELOG.md` complexity is way too much in this package anyway: we don't need it anymore anyway (although we need to keep it for BC) 

See https://github.com/laminas/technical-steering-committee/blob/4a8ca31d5938a242f071ac6c4f607c07bf88ac95/meetings/minutes/2021-02-01-TSC-Minutes.md#drop-changelogmd-from-repositories-when-it-leads-to-merge-conflicts

Reply:
Thanks a rebase from `1.14.x` via git, after retargeting the pr to `1.14.x` on Github, did the trick.